core/src/main/resources/hudson/PluginManager/_table.js

```diff
@@ -10,46 +10,214 @@ function checkPluginsWithoutWarnings() {
+                                htmlDivElement = document.createElement('div');
+                                htmlDivElement.classList.add('plugin-manager__categories');
+                                plugin.categories.forEach(function (category) {
+                                    var htmlAnchorElement = document.createElement('a');
+                                    htmlAnchorElement.href = '?filter=' + category;
+                                    htmlAnchorElement.classList.add('plugin-manager__category-label')
+                                    htmlAnchorElement.text = category
+                                    htmlDivElement.append(htmlAnchorElement)
+                                })
+                                htmlTableDataCellElement.append(htmlDivElement)
+                            }
+                            if (plugin.excerpt) {
+                                htmlDivElement = document.createElement('div');
+                                htmlDivElement.classList.add('excerpt');
+                                htmlDivElement.innerHTML = plugin.excerpt
+                                htmlTableDataCellElement.append(htmlDivElement)
+                            }
+                            htmlTableRowElement.append(htmlTableDataCellElement);
+                            
+                            if (plugin.newerCoreRequired) {
+                                htmlDivElement = document.createElement('div');
+                                htmlDivElement.classList.add('alert', 'alert-danger');
+                                htmlDivElement.innerHTML = plugin.newerCoreRequired
+                                htmlTableDataCellElement.append(htmlDivElement)
+                            }
+
+                            if (plugin.newerJavaRequired) {
+                                htmlDivElement = document.createElement('div');
+                                htmlDivElement.classList.add('alert', 'alert-danger');
+                                htmlDivElement.innerHTML = plugin.newerJavaRequired
+                                htmlTableDataCellElement.append(htmlDivElement)
+                            }
+
+                            if (plugin.dependenciesNewerJava) {
+                                htmlDivElement = document.createElement('div');
+                                htmlDivElement.classList.add('alert', 'alert-danger');
+                                htmlDivElement.innerHTML = plugin.dependenciesNewerJava
+                                htmlTableDataCellElement.append(htmlDivElement)
+                            }
+
+                            if (plugin.unresolvedSecurityWarnings) {
+                                htmlDivElement = document.createElement('div');
+                                htmlDivElement.classList.add('alert', 'alert-danger');
+                                htmlDivElement.innerText = plugin.unresolvedSecurityWarnings.text
```

Comment:
Its probably fine, but i *think* textContent is safer, and from reading online, faster - https://developer.mozilla.org/en-US/docs/Web/API/Node/textContent

Reply:
From a quick read it seems all those differences are when reading it not setting the value but sure can change